MAINTAINERS.md

```diff
@@ -0,0 +1,11 @@
+# Lima maintainers
+
+| Name         | GitHub ID (not Twitter ID)                     | E-mail address for reporting security issues | GPG key ID       |
+|--------------|------------------------------------------------|----------------------------------------------|------------------|
+| Akihiro Suda | [@AkihiroSuda](https://github.com/AkihiroSuda) | (See GitHub profile)                         | 49524C6F9F638F1A |
```

Comment:
Maybe I'm doing it wrong, but searching https://keys.openpgp.org for either `suda.kyoto@gmail.com` or `49524C6F9F638F1A` shows no results.

Reply:
Uploaded